### ISSUE TITLE
Block Search - Group Settings

### ISSUE THREAD

**Issue by Thunder33345** [ImpactDevelopment/ImpactClient#1414]
**Is your feature request related to a problem? Please describe.**
Do you ever just have tons of blocks you want to constantly keep tabs on with search, but sometimes, you dont, so you have to remove a portion of it, only to re-add them one by one
Or just say want a different blocks to have different settings etc?

**Describe the solution you'd like**
A customizable individually togglable group list for a category of blocks you wanna keep tabs on
And to be able to customize each group settings individually

**Additional context**
Basically rather then removing and re-adding, you can just toggle the said list
If no group list is specified, the normal "default" will be used(basically like right now)
probably longterm...
